tests/mentionui.js

```diff
@@ -852,6 +852,44 @@ describe( 'MentionUI', () => {
 			} );
 		} );
 
+		describe.only( 'callback function using data from editor', () => {
```

Comment:
You mistakenly committed this with `.only()` :)

Reply:
Yeah that would explain the coverage drop. ;-) We have eslint rules for these in our project (however using jasmine, not karma)  to avoid these precommit. Thanks for the hint.